src/app/containers/terms-and-conditions/terms-and-conditions.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { TCDocument } from '@hmcts/rpx-xui-common-lib';
+import { Store, select } from '@ngrx/store';
+import { LoadTermsConditions } from 'src/app/store';
+import * as fromRoot from '../../store';
+
+@Component({
+    selector: 'exui-terms-and-conditions',
+    templateUrl: './terms-and-conditions.component.html'
+})
+export class TermsAndConditionsComponent implements OnInit {
+
+    public document: TCDocument = null;
+
+    constructor(private readonly store: Store<fromRoot.State>) {
+    }
+
+    public ngOnInit() {
+        this.store.pipe(
+            select(fromRoot.getTermsAndConditions)
```

Comment:
it needs to complete memory leeks

Reply:
If you mean that we should save a reference to the subscription to unsubscribe in an ngDestroy, then yes, I agree.
If you mean something else, please comment further.